web_development_101/javascript_basics/fundamentals-2.md

```diff
@@ -29,7 +29,7 @@ Depending on what kind of work you're doing, you might end up working more with
 
 1. Read and code along with [yet another MDN tutorial](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Strings) on the topic.
 2. Go through [this lesson](https://www.w3schools.com/js/js_string_methods.asp) to learn a bit more about what you can do with strings... be sure to do the exercises at the end!
-3. Vocabulary time: a __method__ is a bit of functionality that is built into the language or into specific data types.  In the previous lesson you learned a few methods that can be used on strings such as `indexOf` and `search`.  An exhaustive list of methods that can be used on strings can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).
+3. Vocabulary time: a __method__ is a bit of functionality that is built into the language or into specific data types.  An exhaustive list of methods that can be used on strings can be found [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String).
```

Comment:
I'm not sure if it makes sense to link to an exhaustive list of methods for strings without any context.